src/auth/auth0-wrapper.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react"
+import Auth0ClientProvider from '../auth/auth0-client-provider'
+
+const withAuth0 = WrappedComponent => {
+  const Auth0Wrapper = props => {
+    const [isAuthenticated, setAuthenticated] = useState()
+    const [isLoading, setLoading] = useState()
+    const [auth0Client, setAuth0Client] = useState()
+
+    useEffect(() => {
+      const connectAuth0 = async () => {
+        setLoading(true)
```

Comment:
I would've assumed that it wouldn't be possible to read a property from this component before the empty useEffect had returned. It looks like that is not the case though and the useEffect is called after the wrapped component gets the props passed down. Can you help me understand that?

Reply:
I'm not sure I understand your question.  Let's chat about this in the morning.